sdk/python/lib/pulumi/runtime/known_types.py

```diff
@@ -254,6 +267,11 @@ def is_custom_resource(obj: Any) -> bool:
     """
     return _custom_resource_type is not None and isinstance(obj, _custom_resource_type)
 
+def is_stack(obj: Any) -> bool:
+    """
+    Returns true if the given type is an Output, false otherwise.
+    """
+    return _stack_resource_type is not None and isinstance(obj, _stack_resource_type)
```

Comment:
question about `None` answered.

Reply:
why can't we just do this check at the other point?  is it a circularity issue?

Reply:
Not sure what you mean by 'the other point'? I didn't put much thought into this, just pattern matched with what was above for `is_custom_resource`, etc.